demos/mouse/mouse.go

```diff
@@ -27,7 +27,11 @@ func (*DefaultScene) Setup(w *ecs.World) {
 	w.AddSystem(&engo.MouseSystem{})
 	w.AddSystem(&engo.RenderSystem{})
 	w.AddSystem(&ControlSystem{})
+
+	// These are not required, but allow you to move / rotate and still see that it works
 	w.AddSystem(&engo.MouseZoomer{-0.125})
+	w.AddSystem(engo.NewKeyboardScroller(500, engo.W, engo.D, engo.S, engo.A))
```

Comment:
The directions are inverted, W goes down, D goes up A goes right and S goes left for me. Also nothing special (rotation) happens when I hover/click (the hand shows up) What does engo.MouseZoomer do?